### ISSUE TITLE
Chewy::UndefinedUpdateStrategy: Index update strategy is undefined in current context.
Please wrap your code with `Chewy.strategy(:strategy_name) block.`

### ISSUE THREAD

**Issue by tarebyte** [education/classroom#376]
Check full details on Airbrake: [#1562757900403702415](https://github-education.airbrake.io/projects/116727/groups/1562757900403702415)

Backtrace:
```
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/chewy-0.8.2/lib/chewy/strategy/base.rb:17:in update
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/chewy-0.8.2/lib/chewy/type/observe.rb:70:in update_index
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/chewy-0.8.2/lib/chewy/type/observe.rb:28:in block in update_proc
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:446:in instance_exec
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:446:in block in make_lambda
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:263:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:263:in block in simple
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:506:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:506:in block in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:506:in each
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:506:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:92:in __run_callbacks__
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:778:in _run_commit_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/transactions.rb:314:in committed!
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/connection_adapters/abstract/transaction.rb:89:in commit_records
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/connection_adapters/abstract/transaction.rb:153:in commit
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/connection_adapters/abstract/transaction.rb:175:in commit_transaction
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/connection_adapters/abstract/transaction.rb:194:in within_new_transaction
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/connection_adapters/abstract/database_statements.rb:213:in transaction
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/transactions.rb:220:in transaction
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/transactions.rb:348:in with_transaction_returning_status
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.4/lib/active_record/transactions.rb:281:in destroy
[PROJECT_ROOT]/app/jobs/destroy_resource_job.rb:5:in perform
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/execution.rb:32:in block in perform_now
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:117:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:117:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:555:in block (2 levels) in compile
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:498:in block (2 levels) in around
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:343:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:343:in block (2 levels) in simple
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/newrelic_rpm-3.13.1.300/lib/new_relic/agent/instrumentation/active_job.rb:46:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/newrelic_rpm-3.13.1.300/lib/new_relic/agent/instrumentation/active_job.rb:46:in perform
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/newrelic_rpm-3.13.1.300/lib/new_relic/agent/instrumentation/active_job.rb:20:in block (3 levels) in <top (required)>
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:441:in instance_exec
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:441:in block in make_lambda
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:342:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:342:in block in simple
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:497:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:497:in block in around
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:498:in block (2 levels) in around
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:343:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:343:in block (2 levels) in simple
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/i18n-0.7.0/lib/i18n.rb:257:in with_locale
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/translation.rb:7:in block (2 levels) in <module:Translation>
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:441:in instance_exec
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:441:in block in make_lambda
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:342:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:342:in block in simple
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:497:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:497:in block in around
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:498:in block (2 levels) in around
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:343:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:343:in block (2 levels) in simple
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/logging.rb:23:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/logging.rb:23:in block (4 levels) in <module:Logging>
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/notifications.rb:164:in block in instrument
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/notifications/instrumenter.rb:20:in instrument
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/notifications.rb:164:in instrument
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/logging.rb:22:in block (3 levels) in <module:Logging>
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/logging.rb:43:in block in tag_logger
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/tagged_logging.rb:68:in block in tagged
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/tagged_logging.rb:26:in tagged
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/tagged_logging.rb:68:in tagged
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/logging.rb:43:in tag_logger
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/logging.rb:19:in block (2 levels) in <module:Logging>
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:441:in instance_exec
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:441:in block in make_lambda
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:342:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:342:in block in simple
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:497:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:497:in block in around
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:505:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:92:in __run_callbacks__
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:778:in _run_perform_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.4/lib/active_support/callbacks.rb:81:in run_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/execution.rb:31:in perform_now
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/execution.rb:21:in execute
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/activejob-4.2.4/lib/active_job/queue_adapters/sidekiq_adapter.rb:42:in perform
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/processor.rb:75:in execute_job
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/processor.rb:52:in block (2 levels) in process
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/chain.rb:127:in block in invoke
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/newrelic_rpm-3.13.1.300/lib/new_relic/agent/instrumentation/sidekiq.rb:33:in block in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/newrelic_rpm-3.13.1.300/lib/new_relic/agent/instrumentation/controller_instrumentation.rb:362:in perform_action_with_newrelic_trace
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/newrelic_rpm-3.13.1.300/lib/new_relic/agent/instrumentation/sidekiq.rb:29:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/chain.rb:129:in block in invoke
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/server/active_record.rb:6:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/chain.rb:129:in block in invoke
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/server/retry_jobs.rb:74:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/chain.rb:129:in block in invoke
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/server/logging.rb:11:in block in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/logging.rb:30:in with_context
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/server/logging.rb:7:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/chain.rb:129:in block in invoke
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/chain.rb:132:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/middleware/chain.rb:132:in invoke
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/processor.rb:51:in block in process
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/processor.rb:98:in stats
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/sidekiq-3.5.0/lib/sidekiq/processor.rb:50:in process
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/calls.rb:28:in public_send
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/calls.rb:28:in dispatch
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/call/async.rb:7:in dispatch
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/cell.rb:50:in block in dispatch
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/cell.rb:76:in block in task
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/actor.rb:339:in block in task
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/task.rb:44:in block in initialize
[PROJECT_ROOT]/vendor/bundle/ruby/2.2.0/gems/celluloid-0.17.1.2/lib/celluloid/task/fibered.rb:14:in block in create
```

**Comment by tarebyte** [comment]
I can confirm this is an issue, working on a solution.

**Comment by tarebyte** [comment]
Closed by #378
